packages/stardog-language-utils/src/namespaceUtils.ts

```diff
@@ -72,3 +72,31 @@ export const namespaceArrayToObj = (array) =>
       [alias]: prefix,
     };
   }, {});
+
+const escapeSequence = /\\u([a-fA-F0-9]{4})/g;
```

Comment:
Up to `U+10FFFF` is supported, so I believe this should be either the above, or `/\\U([a-fA-F0-9]{8})/g` 
From https://www.w3.org/TR/sparql11-query/ :

19.2 Codepoint Escape Sequences

A SPARQL Query String is processed for codepoint escape sequences before parsing by the grammar defined in EBNF below. The codepoint escape sequences for a SPARQL query string are:

Escape | Unicode code point
-- | --
'\u' HEX HEX HEX HEX | A Unicode code point in the range U+0 to U+FFFF inclusive corresponding to the encoded hexadecimal value.
'\U' HEX HEX HEX HEX HEX HEX HEX HEX | A Unicode code point in the range U+0 to U+10FFFF inclusive corresponding to the encoded hexadecimal value.